ext/liquid_c/tag_markup.h

```diff
@@ -0,0 +1,37 @@
+#ifndef LIQUID_TAG_MARKUP_H
+#define LIQUID_TAG_MARKUP_H
+
+#include "c_buffer.h"
+#include "block.h"
+
+typedef struct tag_markup {
+    uint32_t flags;
+    VALUE tag_name;
+    VALUE markup;
+    VALUE block_body_obj;
+    block_body_t *block_body;
+} tag_markup_t;
+
+typedef struct tag_markup_header {
+    uint32_t flags;
+    uint32_t total_len;
+    uint32_t tag_name_offset;
+    uint32_t tag_name_len;
+    uint32_t markup_offset;
+    uint32_t markup_len;
+    uint32_t block_body_offset;
+} tag_markup_header_t;
+
+#define TAG_FLAG_UNKNOWN (1 << 0)
+#define TAG_UNKNOWN_P(tag) (tag->flags & TAG_FLAG_UNKNOWN)
```

Comment:
I see this flag being set, but I don't see it being used anywhere. Is it needed during deserialization?

Reply:
I think it makes the most sense to add this flag in this PR. It's [used in deserialization](https://github.com/Shopify/liquid-c/pull/138/files#diff-b3e1cbc8064ced465a9e39b531c907a11b39fa015b693b6738139849710b5f9dR364).